ciao-controller/main.go

```diff
@@ -73,6 +91,8 @@ var cephID = flag.String("ceph_id", "", "ceph client id")
 
 var adminSSHKey = ""
 
+var cnciNet cnciNetFlag = "192.168.0.0"
```

Comment:
Should this be specified in CIDR notation? Also is the body of the commit message correct? It says it should be a /18 should it be /16?

Reply:
@rbradford no - we don't need a cidr notation as we require that the network have enough hosts to cover the maximum needed number of cncis - so it just doesn't really need cidr notation. /18 is the max number of cncis we'll ever need.

Reply:
As this IP address range is exposed to the user, i.e. from instances these IPs are reachable I think we should choose a range that is much less widely used than 192.168.0.0/18 as this would prevent instances from accessing resources in the external network if that range is in use.

Reply:
it's just a default - it's configurable by the user via the config file.

Reply:
I will have to make sure that it's documented that they can't chose 172.16-31 and that this is the default.

Reply:
@rbradford and btw just to correct your comment - these IP addresses are not reachable by the instances. This network is for cnci to cnci communication only - and the physical IPs of the CNCI are what would actually go out on the wire. The issue would be if you had the same network for the physical IPs as the cnci tunnel IPs.

Reply:
Before your change any packets that an instance sent to 192.168.0.0/18 would have fallen through the default route and NATing on the CNCI and hit the external network. However after your change those packets will be directed towards the CNCI.

These IP addresses are reachable from instances:
Here is the output from mtr:
```
 Host                                                         Loss%   Snt   Last   Avg  Best  Wrst StDev
 1. 172.16.0.33                                                0.0%     3    0.6   0.5   0.4   0.6   0.0
 2. 192.168.0.4                                                0.0%     2    0.8   0.8   0.8   0.8   0.0
```

My problem is that your default range encompasses the most common RFC1918 subnet 192.168.1.1/24 and its going to be a problem for users.

Please choose a default network which is much less common. Also you should modify ciao-deploy to reflect this new configuration option.

Reply:
ok, I switched it to 192.168.128.0 and added the option to configure to ciao-deploy.